packages/backend/src/modules/tvl2/indexers/BlockTimestampIndexer.ts

```diff
@@ -10,20 +10,19 @@ import { SyncOptimizer } from '../utils/SyncOptimizer'
 
 export type BlockTimestampProvider = EtherscanClient | BlockscoutClient
 
-export interface BlockTimestampIndexerDeps extends ManagedChildIndexerOptions {
+export interface BlockTimestampIndexerDeps
+  extends Omit<ManagedChildIndexerOptions, 'name'> {
   blockTimestampProvider: BlockTimestampProvider
   blockTimestampRepository: BlockTimestampRepository
   chain: string
   syncOptimizer: SyncOptimizer
 }
 
 export class BlockTimestampIndexer extends ManagedChildIndexer {
-  indexerId: string
-
   constructor(private readonly $: BlockTimestampIndexerDeps) {
-    super($)
-    this.$.logger = this.$.logger.for(this)
-    this.indexerId = $.id
+    const logger = $.logger.tag($.chain)
+    const name = 'block_timestamp_indexer'
```

Comment:
You could do the following in ManagedMultiIndexer (or UIF directly). Could be generic and less things to remember when creating indexers.
```suggestion
    // gets name of the parent class and converts `PascalCase` to `snake_case`
    const name = this.constructor.name.split(/\.?(?=[A-Z])/).join('_').toLowerCase()
```